src/grammar.coffee

```diff
@@ -240,6 +240,7 @@ grammar =
     o 'Super'
     o 'This'
     o 'SUPER Arguments',               -> new SuperCall LOC(1)(new Super), $2, no, $1
+    o 'DYNAMIC_IMPORT Arguments',      -> new DynamicImportCall LOC(1)(new DynamicImport), $2
```

Comment:
Technically it looks like dynamic imports are allowed to be object spread (eg `{...import('module')}` compiles in Babel)

Added a corresponding test

Reply:
It evaluates in browsers too:

![image](https://user-images.githubusercontent.com/456802/54472292-4c6cd700-4783-11e9-87f8-2a8fd822b43c.png)

Though it makes no sense, as the `import()` returns a Promise. Not sure how that’s supposed to interact with object spread, and apparently it doesn’t 😄but still the code compiles even if it’s useless.